README.md

```diff
@@ -1,70 +1,59 @@
-
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+- HTML5, CSS3, JS ES6
+- React
+- Webpack
+- Linters: ESLint, Stylelint, Webhint
+- GitFlow
+- Jest
 
-## Learn More
+## Author
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+👤 **Artur Gouveia**
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+- GitHub: @Arturgouveia1970
+- Twitter: @Arturgouveia10
+- LinkedIn: artur-gouveia-323868197
+   
+## Getting Started
 
-### Code Splitting
+To get a local copy up and running follow these simple example steps.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+### Prerequisites
 
-### Analyzing the Bundle Size
+You must have a browser on your device, a visual code, a git version installed, node.js installed in your device.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+### Usage
 
-### Making a Progressive Web App
+Set up the project by cloning and changing into the repo directory:
+https://github.com/Arturgouveia1970/math-magician-3.git
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
 
-### Advanced Configuration
+## 🤝 Contributing
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](../../issues/).
 
-### Deployment
+## Show your support
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+Give a ⭐️ if you like this project!
 
-### `npm run build` fails to minify
+## 📝 License
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+This project is MIT licensed.
+- Page titles
+- Image text alternatives** (<img alt="alt text">)
+- Text headings
+- Color contrast
+- Resize
+- Interaction
+- Moving content
+- Multimedia
+- The basic structure of the page
```

Comment:
- [ ] Nice work applying the changes asked by the previous reviewer but please remove this list of items below the license section because it's not related to your current project to make your readme file more professional.